src/Routes.js

```diff
@@ -1,7 +1,6 @@
 import { Redirect, Route, Switch } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import React, { Suspense, lazy } from 'react';
-import { routes as paths } from '../package.json';
```

Comment:
@kgaikwad @karelhala the reason of doing this change is webpack was giving warning `Should not import the named export` hence did it to fix it.

Reply:
You should not use named export from the package.json. If you need them to move the routes to a different file. Before this data was used for generating site maps. But these were never published and were very unreliable. So it's not used by anything anymore.

Reply:
yes correct @Hyperkid123 , for now I feel passing routes directly is also fine. What you think?

Reply:
also thank you for sharing the details around it 🙌🏻